_src/pl/static/js/checkbox-replace.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
@AlasdairSwan Yes sir. I've written ~~all~~ most of the JS that currently exists in the PL.

I'm assuming we'll need to refactor a bunch of it prior to bringing this work into the platform so it works fluidly. What you currently see is for the PL site specifically, with intentions of "making it work" in the platform when we get there. Does that make sense?